core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -146,64 +146,65 @@ class BrokerTopicMetrics(name: Option[String]) extends KafkaMetricsGroup {
     case Some(topic) => Map("topic" -> topic)
   }
 
+  case class LazyMeter(metricType: String,
```

Comment:
Maybe we should call this `MeterWrapper` or something since it's not necessarily lazy. Nit: both parameters can be on the same line.